src/web/client/common/worker/webworker.js

```diff
@@ -0,0 +1,165 @@
+                },
+            };
+
+            AzureFluidClient._clientInstance = new AzureClient(afrClientProps);
+        }
+        return this._clientInstance;
+    }
+
+    static async fetchContainerAndService(config, id) {
+        if (
+            this._container?.connectionState !== ConnectionState.Connected
+        ) {
+            const azureClient = this.getInstance(config);
+            const { container, services } = await azureClient.getContainer(
+                id,
+                containerSchema
+            );
+            if (container.connectionState !== ConnectionState.Connected) {
+                await new Promise((resolve) => {
+                    container.once("connected", () => {
+                        resolve();
+                    });
+                });
+            }
+            this._container = container;
+            this._audience = services.audience;
+            this._userSharedMap = container.initialObjects.sharedState;
+        }
+        return {
+            container: this._container,
+            audience: this._audience,
+            map: this._userSharedMap,
+        };
+    }
+}
+
+async function loadContainer(config, id, swpId) {
+    try {
+        const { container, audience, map } =
+            await AzureFluidClient.fetchContainerAndService(config, swpId);
+
+        const existingMembers = audience.getMembers();
+
+        // TODO: insert user with entity id in container
+
+        audience.on("memberRemoved", (clientId, member) => {
+            if (!existingMembers.get(member.userId)) {
+                self.postMessage({
+                    type: "member-removed",
+                    userId: member.userId,
+                });
+            }
+        });
+
+        if (!initial) {
+            existingMembers.forEach(async (value, key) => {
+                const otherUser = value;
+                const connectionArray = otherUser.connections;
+
+                const EntityID = [];
+
+                for (let i = 0; i < connectionArray.length; i++) {
+                    const connection = connectionArray[i]?.id;
+                    EntityID.push((await container.initialObjects.sharedState.get('selection').get()).get(connection));
+                }
+
+                self.postMessage({
+                    type: "client-data",
+                    userName: otherUser.userName,
+                    userId: key,
+                    containerId: swpId,
+                    entityId: EntityID,
+                });
+            });
+            initial = true;
+        }
+
+        map.on("valueChanged", async (changed, local) => {
+            if (!local) {
+                const otherUser = map.get(changed.key);
+                const connectionArray = otherUser.connections;
+
+                const EntityID = [];
+
+                for (let i = 0; i < connectionArray.length; i++) {
+                    const connection = connectionArray[i]?.id;
```

Comment:
better naming here - ConnectionId instead of connection

Reply:
done!